examples/nucleo_h723zg/can/main.cpp

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
This example has been compiled but not run. I would appreciate a test run on one of the standard nucleo/discovery boards by someone who owns one.